<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-F0G4AAB" failures="1" tests="2" name="Command line test" time="0.529" errors="0" timestamp="2022-01-15T18:49:46 AMT">
  <testcase classname="e2e.TestNGProject" name="@AfterMethod afterTest" time="0.499">
    <failure type="utils.APIException" message="TestRail API returned HTTP 400(&amp;quot;Field :run_id is not a valid ID.&amp;quot;)">
      <![CDATA[utils.APIException: TestRail API returned HTTP 400("Field :run_id is not a valid ID.")
at utils.APIClient.sendRequest(APIClient.java:197)
at utils.APIClient.sendPost(APIClient.java:64)
at e2e.TestNGProject.afterTest(TestNGProject.java:68)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod afterTest -->
  <testcase classname="e2e.TestNGProject" name="@BeforeMethod beforeTest" time="-1.642258186279E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod beforeTest -->
  <testcase classname="e2e.TestNGProject" name="@AfterMethod afterTest" time="-1.642258186279E9">
    <skipped/>
  </testcase> <!-- @AfterMethod afterTest -->
  <testcase classname="e2e.TestNGProject" name="invalidLogin" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at e2e.TestNGProject.invalidLogin(TestNGProject.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 41 stack frames]]>
    </failure>
  </testcase> <!-- invalidLogin -->
  <testcase classname="e2e.TestNGProject" name="validLogin" time="0.0">
    <skipped/>
  </testcase> <!-- validLogin -->
</testsuite> <!-- Command line test -->
